src/Microsoft.AspNetCore.Build/targets/ProductInfo.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
-        <!-- Product-specific properties -->
-        <DefaultAuthors>Microsoft Open Technologies, Inc.</DefaultAuthors>
-        <ProductStartDate>2015-01-01</ProductStartDate>
-        <BuildQuality>alpha</BuildQuality>
-        <BuildVersion>1.0.1</BuildVersion>
-    </PropertyGroup>
+  <PropertyGroup>
+    <!-- Product-specific properties -->
+    <DefaultAuthors>Microsoft Open Technologies, Inc.</DefaultAuthors>
```

Comment:
That got fixed recently - https://github.com/aspnet/KoreBuild/commit/c9101e20a6c158f38fe8cd32d3024505519c06a1#diff-35569808bff7288226d124655aa58207R30